tests/playbooks/tests_wireless.yml

```diff
@@ -0,0 +1,92 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  vars:
+    interface: wlan0
+  tasks:
+    # mac80211_hwsim only seems to be available and working on RHEL/CentOS
```

Comment:
Added this workaround as `ensure_provider_tests.py` complained if this was in the tests_wireless_nm.yml playbook

Reply:
We can also adjust `ensure_provider_tests.py` to add extra run_conditions to the shim playbook, like adding a comment in the dictionary for NM tests with another key `extra_run_conditions` and then add them to the shim playbook. What do you think?

Reply:
Yeah that might be a less convoluted way of doing this. Maybe something like `platforms` would be better? Could default to `all` but you could specify RHEL7 and CentOS7 for this test?

Reply:
Good point. I think your original suggestion is better, should we create a new issue for this?

Reply:
I added support for this in https://github.com/tyll/network/tree/extra_run_condition - can you add the commit from there to this PR to be able to use it here. I would like to have an actual test that uses it before adding it to the repo.

Reply:
Have added this commit now. Works well :)